spectrum/autism_prevalence_map/views.py

```diff
@@ -3,11 +3,14 @@
 
 from django.shortcuts import render
 from django.http import JsonResponse, HttpResponse
-import os
 from datetime import date
-import re, csv
+import re, csv, os, environ
 from django.contrib.postgres.search import SearchVector, SearchQuery
 
+root = environ.Path(__file__) - 3
+env = environ.Env(DEBUG=(bool, False),)
+environ.Env.read_env()
```

Comment:
I'm not sure why we need Lines 10-12.
Can we just use `DJANGO_ALLOWED_HOSTS`, which we have used before?